docs/containerd-2.0.md

```diff
@@ -0,0 +1,105 @@
+# containerd 2.0
+
+Please try out the release binaries and report any issues at <https://github.com/containerd/containerd/issues/>.
+
+## What's new
+
+### Adds top level configuration migration support
+
+This release adds containerd configuration migration support during daemon startup time. This feature means previous versions of containerd configuration will continue to work on future daemon versions; however, it is recommended to migrate to the latest version (`version = 2`) to avoid migrations and optimize daemon startup time. See [RELEASES.md](../RELEASES.md#daemon-configuration) for full details.
+
+### Adds deprecations warnings to Introspection API
+
+This release adds deprecations warnings to the `ServerResponse` for the introspection service. (`/containerd.services.introspection.v1.Introspection/Server`)
+
+To ease containerd 2.0 transition, the community has moved to backport this feature to containerd 1.6 and 1.7 release branches to give users another tool to identify the sticking points for migrating containerd versions.
+
+Administrators whose workloads are running on containerd versions >= 1.6.25, >= 1.7.9 can query their containerd server to receive  warnings for features marked for deprecation and are in the critical path. With the `ctr` client, users can run `ctr deprecations list`.
+
+### Adds plugin info to Introspection API
+
+This release adds the `PluginInfo` API definition to the introspection service. (`/containerd.services.introspection.v1.Introspection/PluginInfo`)
+
+`PluginInfo` defines the API for returning information directly from a [containerd] plugin [if the plugin supports it].
```

Comment:
I'm not entirely sure how to interpret this without opening the code.  What information is returned by plugins?  An example might help here.

Reply:
Iterated on in https://github.com/containerd/containerd/commit/b6ce8f17f59c4993614d2b214482f20c4c6569a4 to include an example.